package.json

```diff
@@ -16,26 +16,32 @@
     "build": "tsc -p tsconfig.json",
     "dev": "nodemon --exec \"npm start\"",
     "start": "probot run ./lib/index.js",
-    "lint": "standard --fix",
+    "lint": "standard **/*.ts --fix",
     "test": "jest && standard",
```

Comment:
This line should also be updated to use the TypeScript file inclusion pattern:

`"test": "jest && standard **/*.ts",`

Reply:
Good catch! 😄

Reply:
Or `jest && npm run lint`

Reply:
> Or `jest && npm run lint`

@zeke That's a possibility though they are not quite the same (`npm run lint` will actively correct easy-to-fix linting issues, whereas `npm test` will not). Having `npm test` not attempt to fix them is probably the ideal approach, I think, as then it will more clearly point out when easy-to-fix linting issues are committed, e.g. in PRs, as they should fail in CI.

Reply:
I was thinking that, but the existing `npm test` doesn't fix the linting errors. I assume that's the intended functionality.